andela/healthchecks-7-horsemen
# #[151948536] User is notified when a job runs too often.

#### What does this PR do?
* Healthchecks notifies a user when a job runs early than it should, i.e if a job runs before the timeout period elapses with an allowance of a reverse grace period which is set to be equal the grace period.
#### Description of Task to be completed?
*  Adding a new db boolean field to store value for often.
* Adding a function in `api/models.py` that is called and therefore sends and alert when a job is early.
* Making changes to the `api/views.py` so that the ping function incorporates sending an alert when a job is ran early.
* Making changes to email and front-end templates to accomodate visual representation of what is in the backend.
#### How should this be manually tested?
* When logged in as a user, ping a check more than once before the deadline`(timeout period - grace period)`.
* Check for changes on dashboard and integrated channels.
#### What are the relevant pivotal tracker stories?
 * `151948536`
